modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/AndesMessageMetadata.java

```diff
@@ -427,4 +428,36 @@ public void addProperty(String key, Object value) {
     public Object getProperty(String key) {
         return propertyMap.get(key);
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (!(o instanceof AndesMessageMetadata)){
+            return false;
+        }
+        AndesMessageMetadata that = (AndesMessageMetadata) o;
+        return getMessageID() == that.getMessageID();
+    }
+
+    @Override
+    public int hashCode() {
+        return (int) (getMessageID() ^ (getMessageID() >>> 32));
+    }
+
+    /**
+     * true if the expiration time is set to the message
```

Comment:
Method comments and field comments shoul start with a capital letter and and end with a perdiod. Let's correct this in all places.